index.html

```diff
@@ -1,13 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This a portfolio of a full stack web developer">
+    <meta name="keywords" content="porfotlio, software developer, web developer">
+    <meta name="author" content="Julio Quezada">
     <link rel="stylesheet" href="./assets/css/style.css">
     <title>Document</title>
 </head>
+
 <body>
-    <h1 class="makeHeaderRed">My portfolio</h1>
+    <header class="header-container">
+        <div class="logo">
+            <p class="main-logo">John Doe</p>
```

Comment:
[OPTIONAL] 👍🏽 

- I think it will be better if you used your name here as well. 🤗

Reply:
@Favourezeugwa thank you very much for you feedback! I will fix it.
Have a great day!